src/bootstrap-table.js

```diff
@@ -919,6 +919,12 @@ class BootstrapTable {
       const $searchInput = Utils.getSearchInput(this)
 
       handleInputEvent($searchInput)
+
+      if (typeof opts.searchClearSelector === 'string') {
```

Comment:
We already handle the clear of the search somewhere, do we really need an additional event binding ?
Can't we replace the existing selector if the `searchClearSelector` option is set ?

Also does it make sense to restrict it to work only in combination with the `searchSelector`?

Reply:
Oke thx for feedback. I will look into it later so if I can get your proposal working.

Reply:
I guess you can move that part outside of that if, in a new if without that condition.

Reply:
Hi @UtechtDustin I'm not sure what you mean unfortunately. Would you mind updating the PR ?